src/plays/tic-tac-toe/Game.css

```diff
@@ -0,0 +1,33 @@
+.grid {

```

Comment:
I would suggest using a more specific class name. Something like `tic-tac-tow-grid`

Reply:
replaced it with `ttc_grid`.